libs/infra-nest-server/src/lib/bootstrap.ts

```diff
@@ -72,7 +73,10 @@ export const createApp = async (options: RunServerOptions) => {
 
   // Enable validation of request DTOs globally.
   app.useGlobalPipes(
-    new ValidationPipe({ whitelist: true, forbidNonWhitelisted: true }),
+    new ValidationPipe({
+      forbidNonWhitelisted: true,
+      exceptionFactory: (errors: ValidationError[]) => errors.join('\n'),
```

Comment:
What is this doing? Joining error objects?

Reply:
Yeah, the error handler wasn't displaying any messages so I couldn't see why the class-validator was failing. After stringifying the errors, I could take actions based on the error message.